redaxo/src/core/lib/csrf_manager.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/**
+ * @author gharlan
+ *
+ * @package redaxo\core
+ */
+class rex_csrf_manager
+{
+    const PARAM = '_token';
```

Comment:
`_csrf_token`? (tokens gibt es in vielen anwendungen, nicht dass wir hier im namen kollidieren)